release/models/platform/openconfig-platform.yang

```diff
@@ -540,11 +546,13 @@ module openconfig-platform {
     }
 
     leaf model-name {
+      when "../removable = 'true' or ../type = 'CHASSIS'";
```

Comment:
Need to prefix the CHASSIS component type

Reply:
Done, thanks!